test/verify/check-setroubleshoot

```diff
@@ -152,14 +152,32 @@ class TestSelinux(MachineCase):
         if m.image in ["rhel-8-0-distropkg"]: # Introduced in #11789
             return
 
+        b.grant_permissions("clipboardRead", "clipboardWrite")
```

Comment:
Is that only for the headless browser, or does interactive chromium ask for permission as well? If it does that, then IMHO the "failed to copy" can happen in practice and we should handle this somehow in the UI.

Reply:
I could not block copying anyhow - setting clipboard in chrome/ium to Ask/Block and still copying worked. This is mostly there because I paste it into terminal. I'll test it how it works for the copying part for tests.

Reply:
But tests need it, otherwise pasting nor copying works. If I understood it correctly, copying to clipboard does not need permissions when triggered by user action (like in `onClick()`) but reading the clipboard needs these permissions even in such case. (From [here](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard)). Here in selinux we implement only copying to clipboard.

Reply:
Thanks. Perhaps this grant_permissions() call could move down in between copying in the changes dialog and pasting into the terminal?

Reply:
It could be moved closer where it is really needed, but as I wrote, in tests it is needed also for copying the changes - seems that the browser does not recognize our tests clicking on button as a real user action.

Reply:
Ack, thanks.